src/main/kotlin/no/nav/familie/ef/sak/oppgave/OppgaveClient.kt

```diff
@@ -81,6 +84,36 @@ class OppgaveClient(
         pakkUtRespons(respons, uri, "ferdigstillOppgave")
     }
 
+    fun leggOppgaveIMappe(oppgaveId: Long) {
```

Comment:
Burde denne ligge i servicen? 
Får vi egentligen allerede lagt oppgaven i riktig mappe i `opprettOppgave` ? 
med `mappeId = finnAktuellMappe(enhetsnummer, oppgavetype)`

Reply:
Henter nå ut mappeId før oppgave opprettes. La til ny metode `finnHendelseMappeId` for å skille fra `finnAktuellMappe` som finner mappen "godkjenne vedtak"